doc/Journal.tex

```diff
@@ -220,143 +220,113 @@ \section{Background}
-Monte Carlo N-Particle (MCNP) is one method used to simulate energy spectrum.  
-Logan et al.\cite{Logan2017} showed statistical agreement between experimental and simulated DRCs.  
-Thus, this work will use simulated DRCs to evaluate the RSM performance.  
-Instead of the full energy peak (FEP), the DRC is formed by adding all counts above 200keV.  
-This value was chosen as Logan et al. noted discrepancies due to scatter in the enclosing environment for counts below approximately 200keV.
-
-Herein, it is assumed that the geometry is related to the DRC.  
-Qualitative studies of this correlation showed that in general, this assumption is valid with two qualifications.  
-First, a discontinuous geometry results in a continuous DRC.  
-To understand the next qualification, define a cell to be a rectangular prism with a given thickness extending from the centroid of the detector (outside of the Aluminum sleeve) in a given direction.
+Finally, due to manufacturing constraints, each mask angle must have a non-zero thickness.
+
+\subsection{Design Assumptions and Limitations}
+% This paragraph in general has me a bit confused.  I added the "detector-mask-source" geometry as I think this is what was being referred to.   
+It is assumed that the detector-mask-source geometry is related to the DRC and that geometry can be reconstructed using the DRC.
+Qualitative studies of this correlation showed that, in general, this assumption is valid with two qualifications.  
+%I am not sure how the first qualificiation fits into the preceeding statements.  Is the point that in contrast to Fitzgerald's design assumptions?
+First, a discontinuous geometry results in a continuous DRC. 
+% Here, I'd start with stating the qualification, then explaining it. 
+To understand the next qualification, define a cell to be a rectangular prism with a given thickness extending from the centroid of the detector (outside of the aluminum sleeve) in a given direction.
 Since the cells do not have impenetrable walls, particles from one source position enter cells pointed at other positions.  
+% Is it the increase in scatters that leads to a higher number of counts or the reduction in photon energy that leads to a higher interaction probability?  As written, I read it as scatters are preferentially towards the detector, or that there is some multiplication happening, both of which aren't true.
 In fact, this phenomena is one of the desirable characteristics of FitzGerald's design as an increase in scattered particles can increase the total number of counts seen by the detector.  
-Unfortunately, there is a corresponding limit to the spacial resolution.  
+Unfortunately, there is a corresponding limit to the spatial resolution.  
+% Solid angle wrt to what?  It seems like this is stating that neighboring rotation angles can have a high degree of correlation depending on the angle discretization.  This limits the directional resoulution of any design.  Is that correct?
```

Comment:
Yes, I need to separate the two independent ideas.  While the RSM may offer an increase in the total counts, it comes with a limit on the spacial resolution.  If the cells are too small relative to the distance of the source, then neighboring cells may see a response comparable to the cell located between the detector and source.

Reply:
Edited accordingly